clipboard/index.html

```diff
@@ -67,20 +67,32 @@
           el.style.display = 'block';
         }
       }
-      grist.ready();
+      function showData(data) {
+        showError("");
+        document.getElementById('text').value = data;
+        document.getElementById('copy').innerHTML = "Copy Text";
+      }
+      grist.ready({
+        columns: ["Column"]
+      });
       grist.on('message', (e) => {
         if (e.tableId) { tableId = e.tableId; }
       });
-      grist.onRecord(function(record) {
-        delete record.id;
-        var keys = Object.keys(record);
-        if (keys.length > 1) {
-          showError("Please hide all but one of these columns from this widget: " + keys.join(", "));
+      grist.onRecord(function(record, mappings) {
```

Comment:
Is it possible to remove a conditional from this callback by using `mapColumnNames`?